### ISSUE TITLE
Issue with the "Required for conformance" part of Accessibility Requirements mapping

### ISSUE THREAD

**Issue by annethyme** [act-rules/act-rules.github.io#618]
I have filed an issue for the ACT TF about an issue in the ACT Rules Format with the way we talk about "conformance level" in the Accessibility Requirements Mapping section: https://github.com/w3c/wcag-act/issues/384

No matter what is decided there, I still think there's something that needs to be fixed on the ACT Rules CG website too.

As it is now, we list this or similar under Accessibility Requirements for all of our rules (see example on https://act-rules.github.io/rules/97a4e1) : 

> Required for conformance to WCAG 2.0 level A

While this is not wrong, it is also not the whole truth about where it is required for conformance, since WCAG 2.1 includes everything in WCAG 2.0, and Level AA and AAA Conformance also includes Level A success criteria. So in fact, it should be something like this: 

> Required for conformance to WCAG 2.0 and above on level A and above

**Comment by jkodu** [comment]
PR #652 in place. Closing the issue.
